ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -97,6 +97,19 @@ extension BrowserViewController: WKNavigationDelegate {
         hideReaderModeBar(animated: false)
       }
     }
+
+    if let tab = tab(for: webView) {
+      tab.redirectURLs = []
+    }
+    appendToRedirectChain(webView)
+  }
+
+  fileprivate func appendToRedirectChain(_ webView: WKWebView) {
```

Comment:
nit: `appendSourceUrlToRedirectChain`

Reply:
Discussed in DM and `appendUrlToRedirectChain` because this appends both the source URL, and then subsequent intermediate URLs.